1.20.0/schemas/energy/EnergyPlanContract.json

```diff
@@ -8,22 +8,37 @@
       ]
     },
     "controlledLoad": {
-      "$ref": "EnergyPlanControlledLoad.json"
+        "$ref": "EnergyPlanControlledLoad.json"
```

Comment:
Space erroneously introduced?

Reply:
This seems to be perfectly aligned in my editor. Lin1 11 starts at Col 7.